packages/js/src/dashboard/components/problems.js

```diff
@@ -15,6 +15,7 @@ export const Problems = () => {
 	const dismissedProblemsList = useSelect( ( select ) => select( "@yoast/dashboard" ).selectDismissedProblems(), [] );
 
 	const dismissedProblems = dismissedProblemsList.length;
+	const problemCount = problemsList.length;
```

Comment:
Why put this as a variable?
This is:
* only used once (actually, the `counts` prop of AlertsTitle already uses it directly)
* no complicated path
* using memory and using memory cleanup cycles

I suppose due to the one above, that IMO is also not worth it.

```suggestion
```